frontend/src/components/CalendarQalendar.vue

```diff
@@ -394,6 +394,7 @@ watch(route, () => {
       @updated-period="dateChange"
       @updated-mode="modeChange"
       ref="qalendarRef"
+      class="h-auto"
```

Comment:
This makes sure the calendar view on the left doesn't scale-y when the schedule configuration column on the right gets higher (due to addition of custom availabilities e.g.).